content/articles/introduction-to-subnetting/index.md

```diff
@@ -0,0 +1,205 @@
+
+  ![routing](/introduction-to-subnetting/routing.jpg)
+
+  [Image source](https://computernetworkingnotes.com)
+
+IP routing process relies on two types of protocols; 
+Routed protocol and Routing protocol.
+
+**Routed Protocol:** This is a network protocol used to send user data between networks. It carries data such as web traffic, flies, messages, etc.
+
+**Routing Protocol**: A routed protocol finds paths (routes) for the IP and the routed protocol. Routing protocols run on Routers, Firewalls, Network servers with Network Operating Systems.
+
+Routing Protocols and the relating Routed Protocol
+| Routing Protocol   | Routed Protocol  |
+| ---------------- | ----------------- |
+| RIP, IGRP, OSPF, EIGRP | IP |
+| RIP, NLSP, EIGRP  | IPX  |
+| RMTP, AURP, EIGRP | Appletalk  |
+
+Routing is categorized into 3 places; 
+
+i. Static 
+
+ii. Default 
+
+iii. Dynamic
+
+**Static Routing (Non-adaptive):** An administrator assigns the path. It is a route configured by adding to the routing table by an administrator. 
+
+**Default Routing:** This sends packets to IP addresses that do not have exact routes. Using the hop devices, regardless of their network type.
+
+**Dynamic Routing(Adaptive):** A router learns dynamic routing by running routed protocols. It adds information to the routing table from connected users immediately.
+
+#### Advantages and disadvantages of each type of routing
+
+| Type of routing  | Advantages  | Disadvantages  |
+| ---------------- | ----------- | -------------- |
+| Static    | * It has no bandwidth usage between the routers. | * It becomes daunting to handle the chore it is to handle this manually |
+             | * Provides security, only the administrator is allowed to have control over the routing network. | * Lots topography is needed. |
+             | * Has no overhead CPU usage on the router. |
+| Default   | * They establish routes that will never go down | * They require a great deal of CPU power |
+| Dynamic   | * It is easier to configure | * It is less secure compared to static and default routing. |
+            | * It is effective in selecting the best route to a destination network. | It is more expensive in terms of bandwidth and CPU. |
+            
+**Classless Inter-Domain Routing (CIDR):** CIDR is based off on the idea of subnet masks. It makes it possible to create supernets adding to subnets. CIDR helps extend the number of available addresses. The CIDR stores as a suffix in the IP address like so; 201.105.34.7/24 (/24) is the CIDR in this address.
+
+**Broadcast Network:** The last ID of a subnet range used to communicate to the host bits within that network. 255.255.255.255 (Broadcast IP).
+
+### Benefits of Subnetting
+
+- Conservation of IP addresses. 
+- Controls network growth. 
+- Enhances network security. 
+- Easier administration. 
+- Enhances performance and speed.   
+- Reduces network traffic. 
+- It simplifies the network.
+
```

Comment:
I suggest adding a conclusion section to perhaps summarize or let the reader know your final thoughts on the topic you just covered.